service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/ServiceInfo.java

```diff
@@ -3,18 +3,67 @@
  */
 package com.lightbend.lagom.javadsl.api;
 
+import java.util.*;
+
 /**
- * Information for this service.
+ * <p>
+ * Public information for this service. A 3rd party registration could use this to register the service on a
+ * service registry.
+ * </p>
+ * <p>
+ * This info requires a name and a group of locatable services. A locatable service is
+ * a named group of {@link ServiceAcl}s.
+ * </p>
+ * <pre>
+ *     Map<String, List<ServiceAcl>> locatableServices = new HashMap<>();
+ *     List<ServiceAcl> helloAcls = Arrays.asList(
+ *        new ServiceAcl(Optional.of(Method.GET), Optional.of("?/hello/.*")),
+ *        new ServiceAcl(Optional.of(Method.POST), Optional.of("/login"))
+ *        );
+ *     List<ServiceAcl> goodbyeAcls = Arrays.asList(
+ *       new ServiceAcl(Optional.of(Method.POST), Optional.of("/logout/.*")));
+ *
+ *     locatableServices.put("hello-service", helloAcls);
+ *     locatableServices.put("goodbye-service", goodbyeAcls);
+ *
+ *     new ServiceInfo("GreetingService",locatableServices);
+ * </pre>
  */
 public final class ServiceInfo {
 
     private final String serviceName;
 
-    public ServiceInfo(String serviceName) {
+    private final Map<String, List<ServiceAcl>> locatableServices;
+
+    /**
+     * @param serviceName       identifies this service. This is the default id when this service act
+     * @param locatableServices a group of locatable services. This information should be publicized on a Service
+     *                          Registry for either client-side or server-side service discovery.
+     */
+    public ServiceInfo(String serviceName, Map<String, List<ServiceAcl>> locatableServices) {
         this.serviceName = serviceName;
+        this.locatableServices = locatableServices;
+    }
```

Comment:
I finally copied the design from scaladsl. This (String, Map[String,ACL]) tuple is very close to what ConductR models when deploying a bundle.
Unfortunately `ServiceRegistryService` messages incoming Lagom's Dev mode Service registry (`ServiceRegistry`) still are limited to (URI, List[ACL]) tuples when we could have (String, URI, Map[String,ACL]). I'll create an issue to improve the ServiceRegistry.